tests/bluetooth/tester/src/btp_bap.c

```diff
@@ -1412,11 +1518,22 @@ static uint8_t ascs_receiver_start_ready(const void *cmd, uint16_t cmd_len,
 	LOG_DBG("Starting stream %p, ep %u, dir %u", &stream->stream, cp->ase_id,
 		stream->stream.ep->dir);
 
-	err = bt_bap_stream_start(&stream->stream);
-	if (err != 0) {
-		LOG_DBG("Could not start stream: %d", err);
-		return BTP_STATUS_FAILED;
-	}
+	while (true) {
+		err = bt_bap_stream_start(&stream->stream);
+		if (err == -EBUSY) {
+			/* TODO: How to determine if a controller is ready again after
+			 * bt_bap_stream_start? In AC 6(i) tests the PTS sends Receiver Start Ready
+			 * only after all CISes are established.
+			 */
```

Comment:
Good question. In this scenario where we only connect the CIS, we are not writing anything over GATT, but we may need to read the streaming state as it can contain long metadata. 

So a `bt_bap_stream_start` for a sink ASE will connect the CIS, but we (as the unicast client) do not have any way of knowing when the CIS has connected if the server does not go into the streaming state. 

I've created https://github.com/zephyrproject-rtos/zephyr/issues/60216 for this